front/app/components/UI/PasswordInput/PasswordInput.tsx

```diff
@@ -28,23 +30,13 @@ const StyledInput = styled(Input)`
   }
 `;
 
-const EyeIcon = styled(Icon)`
-  width: 22px;
-  height: 15px;
-`;
-
-const EyeClosedIcon = styled(Icon)`
-  width: 22px;
-  height: 19px;
-  margin-bottom: -1px;
-`;
-
-const ShowPasswordButton = styled(Button)`
+const ShowPasswordIconButton = styled(IconButton)<{ showPassword: boolean }>`
```

Comment:
This is a general thought (not necessarily linked to the PR). We still have to restyle these IconButton pretty often. Adding the Box props to the component will solve some that (like here, for example), but I feel a definite need for better standartization so that we don't have to customize so much...

Reply:
Agreed :)